core/templates/dev/head/css/oppia.css

```diff
@@ -65,6 +65,10 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
   background-color: #115FD4;
 }
 
+/* Attributes for material-icons. */
+.material-icons{
```

Comment:
There are other icons with .material-icons classes in the codebase, and some of them already have "vertical-align: middle;" -- see e.g. the site feedback form link in base.html for an example; there are others. Can you please remove the duplicate inline style from these?

Also, lint issue: please add one space before the {.

Thanks!